src/main/resources/component-kernel.conf

```diff
@@ -126,6 +126,8 @@ http.outcall {
     # This timeouts can be expanded by new blocks in dependant projects. For the configuration to take effect, on an
     # outcall the sirius.kernel.xml.Outcall.withConfiguredTimeout needs to be called with the defined block
     # key as parameter.
+    #
+    # NOTE: Please make sure to always define connect- and readTimeout in new blocks to avoid provoking a ConfigException.
```

Comment:
? defaults should work independent of each other, so an extension may only define a connect  timeout or vice versa

Reply:
You're right, i tested the wrong way. It works as it should.